.travis.yml

```diff
@@ -6,7 +6,8 @@ before_install:
   - tar -xf factorio.tar.gz
   - npm install -g codecov
 script:
-  - node build-mod --bump-patch --factorio-version 0.18 --output-dir sharedMods
+  - npx lerna bootstrap --no-ci
```

Comment:
Why would you use --no-ci on CI?

Reply:
As noted in the commit that does that it reqires package-lock.json files to be commited to the repository and we do not for the time being do this.

Reply:
Makes sense, we should maybe consider doing that sometime, but that doesn't belong here.